packages/web-components/fast-components/src/radio/radio.styles.ts

```diff
@@ -0,0 +1,138 @@
+import { css } from "@microsoft/fast-element";
+import { disabledCursor, display } from "../styles";
+import { focusVisible } from "../styles/focus";
+import { SystemColors } from "../styles/system-colors";
+import { heightNumber } from "../styles/size";
+import {
+    neutralFillInputHoverBehavior,
+    neutralFillInputRestBehavior,
+    neutralForegroundRestBehavior,
+    neutralOutlineHoverBehavior,
+    neutralOutlineRestBehavior,
+} from "../styles/recipes";
+
+export const RadioStyles = css`
+    --var
```

Comment:
probably don't need this `--var`

Reply:
that's a global var, surprised you didn't know that Nick. Removing.